server/routes/api/messages.js

```diff
@@ -43,4 +51,88 @@ router.post("/", async (req, res, next) => {
   }
 });
 
+// Update mssage read status in conversation
+router.patch("/", async (req, res, next) => {
```

Comment:
Good job using patch 👍  
However the url should not be on the root of the messages api (`/`) since we're not actually updating the entire resource itself (editing message content), we're just updating the read status, so a more appropriate url would be something like `/read`

Reply:
noted !